lib/stemcell/templates/bootstrap.sh.erb

```diff
@@ -226,7 +232,9 @@ GIT_SSH=${git_wrapper} git fetch
 git reset --hard origin/\${branch}
 git clean -fdx
 
-json="{\"role\": \"\${role}\", \"env\": \"\${env}\", \"branch\": \"\${branch}\"}"
+base_json="{\"role\": \"\${role}\", \"env\": \"\${env}\", \"branch\": \"\${branch}\"}"
+json=\`echo \$node_json \$base_json | jq -s '.[0] + .[1]'\`
```

Comment:
there's no reason to involve an external binary; you can just construct the variable you want in ruby and pass it, fully constructed, into the template.

Reply:
Alright, I've added a commit which moves this over to 'ruby-land'.